frontend/pages/queries/edit/EditQueryPage.tsx

```diff
@@ -5,7 +5,12 @@ import { InjectedRouter, Params } from "react-router/lib/Router";
 
 import { AppContext } from "context/app";
 import { QueryContext } from "context/query";
-import { DEFAULT_QUERY, DOCUMENT_TITLE_SUFFIX } from "utilities/constants";
+import {
+  DEFAULT_QUERY,
+  DOCUMENT_TITLE_SUFFIX,
+  INVALID_PLATFORMS_FLASH_MESSAGE,
+  INVALID_PLATFORMS_REASON,
```

Comment:
are these only used on the query pages? if so we may want to have them defines in a file closer instead of in the global constants

Reply:
Yes. Leaving as is per conversation.